### PR TITLE
Feat: added variations of button 

### PR DESCRIPTION
**Pull request by anamkaa**
Screenshot (224)
Screenshot (225)

https://user-images.githubusercontent.com/75376715/152120990-53ca5ea3-f665-4d81-89c2-e165467c04ea.mp4

### CODE REVIEW

**Review by aishwarya-bm — COMMENTED** [review]
Lot of variations,looks great! 
Good to merge!

File: stylesheets/components/button.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  background-color: var(--bg-white);
+}
+
+.btn-dark {
+  background-color: black;
+  color: white;
+  border: 1px solid black;
+}
+
+.btn-lg {
```
Comment (vivekmehto) [inline stylesheets/components/button.css:20]: Don't give a fixed font size. You can use rems.

File: stylesheets/abstract/variables.css [hunk 2]
```diff
@@ -2,16 +2,27 @@
...
-    --warning:#ffea00;
-    --warning-bg:#fffde6;
+    --warning:#ff6d00;
+    --warning-bg:#ffd3b3;
+    --warning-bg-light: #ffe2cc ;
 
     --error:#ff002b;
-    --error-bg:#ffe6ea;
-    
+    --error-bg: #ffccd5;
```
Comment (vivekmehto) [inline stylesheets/abstract/variables.css:22]: You can use or add more friendly color names for the user. just a suggestion.

**Review by vivekmehto — COMMENTED** [review]
Everything looks good apart from the given suggestions.

File: stylesheets/abstract/variables.css (hunk 2 above)
Comment (anamkaa) [reply stylesheets/abstract/variables.css:22]: Sure, will try to add more.

File: stylesheets/components/button.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/components/button.css:20]: Thanks , will keep that in mind.

**Pull request merged**